spec/javascripts/details/views/notification_bar_spec.js.coffee

```diff
@@ -13,30 +14,119 @@ describe 'Notification', ->
       expect(notification instanceof Notifications).to.eql(true)
   
   describe '#_postNotification', ->
-    it "create a notification", ->
+    it "create a notification without type", ->
```

Comment:
Prefer naming of individual specs to read like a sentence including the `it`, so "it create**s** a notification without type" (note the S)